sagemaker-inference-recommender/auto-scaling/optimize_endpoint_scaling.ipynb

```diff
@@ -0,0 +1,1018 @@
+{
```

Comment:
<p>Break up sentences: "The first and the recommended option is TragetTracking, in this option, a target value that represents the average utilization or/and throughput of single host is set as a scaling threshold." -&gt; "The first and the recommended option is TragetTracking. In this option, a target value that represents the average utilization or/and throughput of single host is set as a scaling threshold. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>